lib/openxml/elements/paragraph.rb

```diff
@@ -0,0 +1,8 @@
+module OpenXml
+  module Elements
+    class Paragraph < OpenXml::Container
+      namespace :a
```

Comment:
This one — and the run — are in the `a` namespace — isn't that the DrawingML namespace? Should these elements go there?

What do you think, @kobsy?